app/views/templates/search/channels_landing.rb

```diff
@@ -56,486 +34,482 @@ def navigation
+            items: [
               {
-                :title =>"Erlkonig",
-                :url => "urlhere",
-                :url => root_url + "channels/music?q=erlkonig",
-                :count => "12",
-                :media_type =>"audio recordings"
+                title: "Erlkonig",
+                url: "urlhere",
+                url: root_url + "channels/music?q=erlkonig",
+                count: "12",
+                media_type: "audio recordings"
               },
               {
-                :title => "Mahler 5",
-                :url => root_url + "channels/music?q=mahler",
-                :count =>  "15",
-                :media_type => "video recordings"
+                title: "Mahler 5",
+                url: root_url + "channels/music?q=mahler",
+                count: "15",
+                media_type: "video recordings"
               },
               {
-                :title =>"Beethoven’s handwriting",
-                :url => root_url + "channels/music?q=beethoven+writing",
-                :count => "2",
-                :media_type =>"scores"
+                title: "Beethoven’s handwriting",
+                url: root_url + "channels/music?q=beethoven+writing",
+                count: "2",
+                media_type: "scores"
               },
               {
-                :title =>"Haitink in Berlin",
-                :url => root_url + "channels/music?q=Haitink",
-                :count => "12",
-                :media_type =>"letters"
+                title: "Haitink in Berlin",
+                url: root_url + "channels/music?q=Haitink",
+                count: "12",
+                media_type: "letters"
               },
               {
-                :title =>"Papageno costumes",
-                :url => root_url + "channels/music?q=Papageno+costumes",
-                :count => "42",
-                :media_type =>"images"
+                title: "Papageno costumes",
+                url: root_url + "channels/music?q=Papageno+costumes",
+                count: "42",
+                media_type: "images"
               },
               {
-                :title =>"Baroque wedding music",
-                :url => root_url + "channels/music?q=Baroque+wedding",
-                :count => "5",
-                :media_type =>"audio recordings"
+                title: "Baroque wedding music",
+                url: root_url + "channels/music?q=Baroque+wedding",
+                count: "5",
+                media_type: "audio recordings"
               }
             ]
           },
-          :promoted =>{
-            :title => "Promoted title",
-            :items =>[
+          promoted: {
+            title: "Promoted title",
+            items: [
               {
-                :title =>"The legacy of Punk",
-                :url => "urlhere",
-                :is_exhibition => true,
-                :bg_image =>"sample/thumb-music.jpg"
+                title: "The legacy of Punk",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.